### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by jonas-bl** [community/community#63429] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Github Desktop not working to sign in...

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ArvedLukas** [comment]
Try to reinstall please

**Comment by queenofcorgis** [comment]
@jonas-bl @ArvedLukas GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
